examples/chunking.ddl

```diff
@@ -0,0 +1,43 @@
+// chunking example
+// https://github.com/HazyResearch/deepdive/tree/master/examples/chunking
+
+words_raw(
+  word_id bigserial,
+  word text,
+  pos text,
+  tag text).
+
+words(
+  sent_id bigint,
+  word_id bigint,
+  word text,
+  pos text,
+  true_tag text).
+
+word_features(
+  word_id bigint,
+  feature text).
+
+tag?(word_id bigint) Categorical(13).
+
+function ext_training 
+  over like words_raw
+  returns like words
+  implementation "/udf/ext_training.py" handles tsv lines.
```

Comment:
Can we remove the leading `/` in the udf paths from ddlog examples and add `${APP_HOME}/` from the compiler?